ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -103,6 +130,11 @@ class TabBarItemView: UIControl {
 			container.widthAnchor.constraint(lessThanOrEqualTo: widthAnchor)
 		])
 
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(themeDidChange),
+                                               name: .didChangeTheme,
+                                               object: nil)
```

Comment:
If the TabBarView is in charge of updating the TabBarItemView's tokens, I'm not sure you need this observer on both. I think having the TabBarItemView update all the tokenized values after the TabBarView sets the new tokens should be enough.